tests/cms/views/view_config.py

```diff
@@ -161,10 +162,10 @@
             ),
             ("events", ROLES),
             ("events_archived", ROLES),
-            ("new_event", ROLES),
+            ("new_event", ROLES - [OBSERVER]),
```

Comment:
Hmm, I find this syntax not ideal, could you add a new constant, maybe `WRITE_ROLES` or something with all roles which are allowed to write contents?

Reply:
OK, I"ll do so :)